webapp/src/components/user_attribute/index.js

```diff
@@ -11,6 +11,7 @@ const REDUCER = `plugins-${pluginId}`;
 function mapStateToProps(state, ownProps) {
     const id = ownProps.user ? ownProps.user.id : '';
     const data = state[REDUCER].attributes[id];
+    const fromWebhook = ownProps.overwriteName && ownProps.overwriteIcon;
```

Comment:
Should the icon be hidden if only one of those two is set?

Reply:
In this case, we could do that. In case the name has been overridden, then we can short-circuit the evaluation there, and then check for the icon. On Mattermost instances with the plugin enabled, I have noticed only cases where both have been overridden for the integration. If that is the case, then we can use one of the two properties to evaluate. We change this to 
```js
const fromWebhook = ownProps.overwriteName || ownProps.overwriteIcon;
```